graylog2-web-interface/src/components/event-definitions/replay-search/EventInfoBar.tsx

```diff
@@ -0,0 +1,218 @@
+      const color = rule.color as StaticColor;
+      const expr = conditionToExprMapper?.[condition];
+
+      if (expr) {
+        const key = `${field}${expr}${value}`;
+
+        if (aggregationsSet.has(key)) {
+          res[key] = color.color;
+        }
+      }
+
+      return res;
+    }, {});
+  }, [aggregations, highlightingRules]);
+
+  return (
+    <FlatContentRow>
+      <Header>
+        <Button bsStyle="link" className="btn-text" bsSize="xsmall" onClick={toggleOpen}>
+          <Icon name={`caret-${open ? 'down' : 'right'}`} />&nbsp;
+          {open ? 'Less details' : 'More details'}
+        </Button>
+      </Header>
+      {open && (
+      <Container>
+        <Row>
+          {!isEventDefinition && (
+          <Item>
+            <b>Timestamp:</b>
+            <Timestamp dateTime={eventData.timestamp} />
+          </Item>
+          )}
+          {isEDUpdatedAfterEvent && (
+          <Item>
+            <b>Event definition updated at:</b>
+            <AlertTimestamp dateTime={eventDefinition.updated_at} />
+            <HoverForHelp iconSize="xs">{
+              `Event definition ${eventDefinition.title} was edited after this event happened.
+              Some of aggregations widgets might not be representative for this event.`
+            }
+            </HoverForHelp>
+          </Item>
+          )}
+          {!isEventDefinition && (
+          <Item>
+            <b>Event definition:</b>
+            <span>
+              <Link target="_blank"
+                    to={Routes.ALERTS.DEFINITIONS.show(eventDefinition.id)}>{eventDefinition.title}
+              </Link>
+            </span>
+          </Item>
+          )}
+          <Item>
+            <b>Priority: </b>
+            <span>{lodash.upperFirst(EventDefinitionPriorityEnum.properties[eventDefinition.priority].name)}</span>
+          </Item>
+          <Item>
+            <b>Execute search every:</b>
+            <span>{executeEvery.duration} {executeEvery.unit.toLowerCase()}</span>
+          </Item>
+          <Item>
+            <b>Search within:</b>
+            <span>{searchWithin.duration} {searchWithin.unit.toLowerCase()}</span>
+          </Item>
+          <Item>
+            <b>Description:</b>
+            <span>{eventDefinition.description}</span>
+          </Item>
+          <Item>
+            <b>Notifications:</b>
+            <span>
+              {
+                notificationList.map(({ id, title }, index) => {
+                  const prefix = index > 0 ? ', ' : '';
+
+                  return (
+                    <>
+                      {prefix}
+                      <Link target="_blank" to={Routes.ALERTS.NOTIFICATIONS.show(id)}>{title}</Link>
+                    </>
+                  );
+                })
+              }
+            </span>
+          </Item>
+          {Object.values(highlightingColors).length && (
+          <Item>
+            <b>Aggregation conditions:</b>
```

Comment:
What do you think about displaying the highlighting colors like we do it in the sidebar?

![image](https://user-images.githubusercontent.com/46300478/221232823-6cb34b0d-8f52-4e93-b6dc-819fbd575045.png)

![image](https://user-images.githubusercontent.com/46300478/221232873-39d2751a-ba24-4a2c-95e1-b9ecacbfcad4.png)

This would unify the UI and it would improve the contrast.